app/api/resources/mentorship_relation.py

```diff
@@ -226,15 +226,15 @@ def put(cls, request_id):
         # if it is an integer
 
         user_id = get_jwt_identity()
-        response = DAO.accept_request(user_id=user_id, request_id=request_id)
+        response = DAO.accept_request(user_id=user_id, relation_id=relation_id)
 
         if response[1] == HTTPStatus.OK:
-            send_email_mentorship_relation_accepted(request_id)
+            send_email_mentorship_relation_accepted(relation_id)
 
         return response
 
 
-@mentorship_relation_ns.route("mentorship_relation/<int:request_id>/reject")
+@mentorship_relation_ns.route("mentorship_relation/<int:relation_id>/reject")
```

Comment:
No need to do any of the changes(i.e from request_id to relation_id) in this class. Since, we still haven't accepted the relation request.